packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -13,6 +13,8 @@ interface CascaderProps {
   separator?: string;
   placeholder?: string;
   options: CascaderOption[];
+  /** Changes the value for every selection, including branch nodes */
```

Comment:
Super small nit, maybe worth mentioning default value?
```suggestion
  /** Changes the value for every selection, including branch nodes. Defaults to true. */
```

Reply:
Absolutely!